modules/developer-cli-odo-switching-between-ephemeral-and-persistent-storage.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// *cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/working-with-storage.adoc
+
+[id="switching-between-ephemeral-and-persistent-storage_{context}"]
+= Switching between ephemeral and persistent storage
+
+You can switch between ephemeral and persistent storage in your project by using the `odo preference` command. `odo preference` modifies the global preference in your cluster.
+
+When persistent storage is enabled, the cluster stores the information between the restarts. 
+
+When ephemeral storage is enabled, the cluster stores temporary information that does not need to persist between the restarts.
```

Comment:
Between this line and the one above, we have classified "information" vs. "temporary information". It's more like, odo treats stuff as temporary when using ephemeral and as not temporary when not using ephemeral. IMO, we could simply modify the statement for ephemeral storage to below:
`When ephemeral storage is enabled, the cluster does not store/persist the information between the restarts.`